drools-wb-services/drools-wb-verifier/drools-wb-verifier-guided-decision-table-adapter/src/test/java/org/drools/workbench/services/verifier/plugin/client/AnalyzerUpdateTestBase.java

```diff
@@ -79,6 +80,23 @@ public ValueSetter setCoordinate() {
         return new ValueSetter();
     }
 
+    protected void setValue(final int rowIndex,
+                            final int columnIndex,
+                            final Date value) {
+        DTCellValue52 dtCellValue52 = table52.getData()
+                .get(rowIndex)
+                .get(columnIndex);
+        dtCellValue52
+                .setDateValue(value);
+        try {
+            updateManager.update(table52,
+                                 getUpdates(rowIndex,
+                                            columnIndex));
+        } catch (UpdateException e) {
```

Comment:
This _smells_ bad.. what was your reasoning to catch `UpdateException`?

Reply:
I removed the UpdateException. I think it used to catch illegal states, but now it only checked if the cell can be found.